src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php

```diff
@@ -61,10 +87,11 @@ private function getCountryCodes(array $countries)
 
         /* @var CountryInterface $country */
         foreach ($countries as $country) {
-            $countryCodes[$country->getCode()] = Intl::getRegionBundle()->getCountryName($country->getCode());
+            $countryName = Intl::getRegionBundle()->getCountryName($country->getCode());
+            $countryCodes[$countryName] = $country->getCode();
```

Comment:
[It looks so](http://symfony.com/doc/current/reference/forms/types/choice.html#choices-as-values), but you can use `choice_label` / `choice_value` options to manipulate the choices.